### PR TITLE
Refactored ChartUtils method into CGPoint extension

### PR DESCRIPTION
**Pull request by jjatie**

### CODE REVIEW

**Comment by liuxuan30** [comment]
Can you tag this PR with project 'More Swift'? On the right panel

**Comment by jjatie** [comment]
I'm still unable to. This is for master. I will PR into 4.0.0 branch if it is for 4.0.

**Comment by liuxuan30** [comment]
don't know Daniel cannot see the gitter. Let's wait a few days;

**Comment by liuxuan30** [comment]
conflict:) I'm taking target for master first

File: Source/Charts/Utils/ChartUtils.swift [hunk 1]
```diff
@@ -81,18 +77,17 @@ extension Double {
     }
 }
 
+extension CGPoint {
```
Comment (liuxuan30) [inline Source/Charts/Utils/ChartUtils.swift]: code style

File: Source/Charts/Utils/ChartUtils.swift [hunk 2]
```diff
@@ -81,18 +77,17 @@ extension Double {
...
+    /// Calculates the position around a center point, depending on the distance from the center, and the angle of the position around the center.
+    func moving(distance: CGFloat, atAngle angle: CGFloat) -> CGPoint {
```
Comment (liuxuan30) [inline Source/Charts/Utils/ChartUtils.swift]: { code style

File: Source/Charts/Utils/ChartUtils.swift [hunk 3]
```diff
@@ -12,10 +12,6 @@
 import Foundation
 import CoreGraphics
 
-#if !os(OSX)
```
Comment (liuxuan30) [inline Source/Charts/Utils/ChartUtils.swift]: why delete this?

Comment (jjatie) [reply Source/Charts/Utils/ChartUtils.swift]: UIKit is not used in the file.

**Comment by jjatie** [comment]
Codestyle fixed

**Comment by liuxuan30** [comment]
Great, all test green.

**Comment by liuxuan30** [comment]
@jjatie @danielgindi sorry to dig this up. Since we finished https://github.com/danielgindi/Charts/projects/4 now, could we begin the process to merge 4.0 into master now? Master and 4.0 has a big difference

**Pull request merged**